source/templates/search.liquid

```diff
@@ -1,8 +1,8 @@
 <h1>Search</h1>
 
-<form action="/search" method="get" class="search form" role="search">
+<form action="/search" method="get" class="search form">
 
-  <div class="form__group">
+  <div class="form__group" role="search">
```

Comment:
I mentioned the difference in my approach in the commit message.  But again, having the `role="search"` on the `<form>` element can negate the inherit role of said form.  So if a user is attempting to navigate a page via forms, in certain screen readers that form will not be revealed and the expected functionality of that screen reader, in forms mode, may not be initiated.  Setting the role="search" to the child div keeps the form semantics in tact, and still provides the extended search landmark discoverability.  This is nothing but an improvement.